tui/src/ui/mod.rs

```diff
@@ -159,17 +159,29 @@ impl UI {
         {
             let mut system = System::new();
             system.refresh_all();
+            let mut target = None;
+            let mut clients = 0;
             for proc in system.processes().values() {
-                let Some(exe) = proc.exe().map(|v| v.display().to_string()) else {
+                let exe = proc.name().to_str().unwrap();
```

Comment:
Preferably, this should not panic and instead just skip this process, please use something like `let Some(exe) = ` again like before the patch.